src/textdescriptives/components/quality_data_classes.py

```diff
@@ -151,6 +153,11 @@ class Config:
         + r"are contained within a duplicate for 2-grams, 18% for 3-grams and 16% "
         + "for 4-grams.",
     )
+    oov_ratio: Interval = Field(
+        (None, 0.2),
+        description="A range for the out-of-vocabulary ratio. Default: (None, 0.2)"
+        + r" i.e. no lower limit, but at most 20% of words are out-of-vocabulary.",
+    )
```

Comment:
seems reasonable, we should test this out on dagw to see if it actually holds.

Reply:
Agreed!

Reply:
Should we do this before we merge this in

Reply:
0.2 seems very conservative to me so I wouldn't say it's necessary before merging in. Should in all likelihood be tuned for the domain in any case

Reply:
Let us create a issue on this an merge it in